Introducing WIF in Secret Sync for Azure Destination (#1816)
hashicorp/web-unified-docs: content/vault/v2.x (rc)/content/docs/sync/azurekv.mdx

Prerequisites:
* Ability to read or create KVv2 secrets
* Ability to create Azure AD user credentials with access to an Azure Key Vault, or configure workload identity federation (WIF)
* Ability to create sync destinations and associations on your Vault server

## Setup

1. If you do not already have an Azure Key Vault instance, navigate to the Azure Portal to create a new
  [Key Vault](https://learn.microsoft.com/en-us/azure/key-vault/general/quick-create-portal).

1. Configure authentication credentials for Azure Key Vault. You can use one of the following methods:

>    - **Client secret authentication**: Create a service principal with a client ID and client secret. This [guide](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal) walks you through creating the service principal.

Reviewer: ```suggestion
   - **Client secret authentication**: Follow Azure instructions to
     [create a service principal](https://learn.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal)
     the use the service principle with a client ID and client secret.
```
Style correction: use meaningful anchor text, avoid "this" as a pronoun

>    - **Workload identity federation (WIF)**: Configure a trust relationship between Vault and Azure. This method eliminates the need to manage long-lived client secrets. See the [WIF setup](#workload-identity-federation-wif) section for details.

Reviewer: ```suggestion
   - **Workload identity federation (WIF)**:
     [Configure a trust relationship]](#workload-identity-federation-wif)
     between Vault and Azure. Use WIF to eliminate the need to manage long-lived
     client secrets.
```

Author: Thanks! Updated the WIF description to use active voice and more descriptive link text.

counterpart. Creating a new secret version in Vault will create a new version in Azure Key Vault. Deleting the secret
or the association in Vault will delete the secret in your Azure Key Vault as well.

## Workload identity federation (WIF)

> Workload identity federation (WIF) lets the Azure Key Vault sync destination authenticate to Azure without managing long-lived client secrets. Instead, Vault exchanges short-lived identity tokens for Azure access tokens.

Reviewer: ```suggestion
Workload identity federation (WIF) lets the Azure Key Vault sync destination
authenticate to Azure by having Vault exchange short-lived identity tokens for
Azure access tokens instead of managing long-lived client secrets.
```
I had trouble parsing the original sentence. Let me know if this edit changed the meaning in some way

Author: Thanks! The revised sentence reads more clearly, and it keeps the original meaning intact.

> Workload identity federation (WIF) lets the Azure Key Vault sync destination authenticate to Azure without managing long-lived client secrets. Instead, Vault exchanges short-lived plugin identity tokens for Azure access tokens.

Reviewer 2: Maybe we can omit the word `plugin` and just say identity tokens

Reviewer 2: Same applies to other places.
